frontend/lib/src/components/elements/Toast/Toast.tsx

```diff
@@ -15,14 +15,15 @@
  */
 
 import React, {
+  memo,
   ReactElement,
   useCallback,
   useEffect,
   useMemo,
   useState,
 } from "react"
 
-import { withTheme } from "@emotion/react"
```

Comment:
**suggestion (non-blocking):** Since we're leaning into hooks, it would be great to add an eslint rule that prevents usage of `withTheme` and suggests using `useTheme` instead. Totally fine if this is a follow-up, since it's not exactly the point of this PR.

Reply:
Sounds good 👍 I added a rule to prevent usage of `withTheme`.